inference-engine/src/inference_engine/src/ie_core.cpp

```diff
@@ -68,10 +68,12 @@ Parsed<T> parseDeviceNameIntoConfig(const std::string& deviceName, const std::ma
         if (deviceName.find("AUTO:") == 0) {
             config_[InferenceEngine::MultiDeviceConfigParams::KEY_MULTI_DEVICE_PRIORITIES] =
                 deviceName.substr(std::string("AUTO:").size());
+            config_.insert({CONFIG_KEY_INTERNAL(MULTI_WORK_MODE_AS_AUTO), InferenceEngine::PluginConfigParams::YES});
```

Comment:
it's not needed since AUTO has this value by default.

Reply:
fixed